### PR TITLE
RTL support in scrollEnabled Tabbar [Android]

### PR DESCRIPTION
**Pull request by alikazemkhanloo**
### Motivation
Tabbar with scrollEnabled not working properly in RTL mode.

- [x] scroll to correct tab 
- [x] move the indicator to the correct location
- [x] add platform specific. as this problem differs in Android and iOS.
### Test plan
use in RTL mode.

### CODE REVIEW

File: src/TabBar.js [hunk 1]
```diff
@@ -190,10 +191,7 @@ export default class TabBar<T: Route> extends React.Component<Props<T>, State> {
 
       this._scrollView &&
         this._scrollView.scrollTo({
-          x: this._normalizeScrollValue(
-            this.props,
-            this._getScrollAmount(this.props, value)
```
[the remark this answers is not in the document]
Comment (satya164) [reply src/TabBar.js]: @alikazemkhanloo yea seems redundant

**Comment by alikazemkhanloo** [comment]
looks like this only happens in Android. I am using version 1.3.2 myself, because of the low performance of new version (at least in RTL and in my phone). so continuing this PR is not easy for me. Any help is appreciated.

**Comment by alikazemkhanloo** [comment]
now it should work.

**Comment by satya164** [comment]
Thanks for the PR. Is it supposed to work on Android or only iOS? I just tested and it works on iOS, but not on Android.

**Comment by alikazemkhanloo** [comment]
What is the issue? It is working in my project, perfectly. actually, I'm using this branch in my app.

**Pull request closed without merging**